python/Ganga/Runtime/bootstrap.py

```diff
@@ -181,6 +181,10 @@ def manualExportToGPI(my_interface=None):
     from Ganga.GPIDev.Lib.Registry.JobRegistry import jobSlice
     exportToInterface(my_interface, "jobSlice", jobSlice, "Functions")
 
+    """Needed for the JobTime if we plan to use the interface to eval the XML as well"""
+    import datetime
+    exportToInterface(my_interface, "datetime", datetime, 'Classes')
```

Comment:
I don't think that we should be using the GPI to evaluate strings in the XML. Looking up the names in the plugin system is how it should be done. The fact that we have to export `datetime` here is a string sign that there is something wrong.